Formula/subversion@1.8.rb

```diff
@@ -16,6 +16,8 @@ class SubversionAT18 < Formula
 
   keg_only :versioned_formula
 
+  deprecate! date: "2020-12-21", because: :versioned_formula
```

Comment:
I wonder if we make this `:does_not_build` since `scons` is on Python 3 but this formula needs Python 2, making it impossible to build in Homebrew.